deploy/kubernetes/overlays/prow-gke-release-staging-head/increase-sidecar-operation-timeout.yaml

```diff
@@ -0,0 +1,8 @@
+# Increase operation timeout for provisioner sidecar.
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: "--timeout=250s"
+# Increase operation timeout for attacher sidecar.
+- op: add
+  path: /spec/template/spec/containers/1/args/-
+  value: "--timeout=250s"
```

Comment:
Do we also need to consider raising the timeout for resizer and snapshotter too?

Reply:
Adding snapshotter timeout as per discussions. The resizer timeouts are fine.